webpack/assets/javascripts/react_app/components/common/forms/ForemanForm/ForemanForm.stories.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { storiesOf } from '@storybook/react';
+import PropTypes from 'prop-types';
+
+import ForemanForm from '.';
+import TextField from '../TextField';
+import RadioButtonGroup from '../RadioButtonGroup';
+import Story from '../../../../../../../stories/components/Story';
+
+const DisplayFormikState = props => (
+  <div style={{ margin: '1rem 0' }}>
+    <pre
+      style={{
+        background: '#f6f8fa',
+        padding: '.5rem',
+      }}
+    >
+      <strong>props</strong> = {JSON.stringify(props, null, 2)}
```

Comment:
I was surprised that my live form values were showing up here in "props," since form state is usually kept in React component "state" and not "props."  But "props" here refers to the props passed down by Formik to your form, I guess?

Reply:
Yes, these are passed down by Formik.